### ISSUE TITLE
AttributeError: 'Properties' object has no attribute '_Properties__stream'

### ISSUE THREAD

**Issue by piskvorky** [TeamMsgExtractor/msg-extractor#303]
**Bug Metadata**

* Version of extract_msg: 0.36.1
* Your python version: Python 3.8
* How did you launch extract_msg?
  - [ ] My command line or
  - [x] I used the extract_msg package

**Describe the bug**

Calling `extract_msg.openMsg(bytes)` leads to the traceback below. Cannot share the raw MSG unfortunately.

**Traceback**

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/extract_msg/message_base.py", line 1030, in header
    return self._header
AttributeError: 'Message' object has no attribute '_header'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/extract_msg/message_base.py", line 941, in date
    return self._date
AttributeError: 'Message' object has no attribute '_date'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/extract_msg/properties.py", line 90, in get
    return self.__props[name]
KeyError: '0E070003'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/extract_msg/utils.py", line 617, in openMsg
    return Message(path, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/extract_msg/message_base.py", line 95, in __init__
    self.header
  File "/usr/local/lib/python3.8/dist-packages/extract_msg/message_base.py", line 1035, in header
    self._header['date'] = self.date
  File "/usr/local/lib/python3.8/dist-packages/extract_msg/message_base.py", line 943, in date
    self._date = self._prop.date if self.isSent else None
  File "/usr/local/lib/python3.8/dist-packages/extract_msg/message_base.py", line 1194, in isSent
    if not self.props.get('0E070003'):
  File "/usr/local/lib/python3.8/dist-packages/extract_msg/properties.py", line 94, in get
    logger.debug(properHex(self.__stream))
AttributeError: 'Properties' object has no attribute '_Properties__stream'
```

**Comment by TheElementalOfDestruction** [comment]
Looks like this was simply caused by one of the internal variables being changed (to better match with other parts of the code) but I ended up missing a spot. I'll double check that this didn't happen anywhere else and add a fix in the next version.

**Comment by TheElementalOfDestruction** [comment]
Fixed in version 0.37.0
